md/licenses.md

```diff
@@ -49,12 +49,25 @@ Then fill in the details in the form, copy the request key in the *Request Key*
 
 The license file will be sent to you by email.
 
-## Install the license
-
-When you receive the license file (`.lic` file extension):
-- If this is the first time you start the bundle, copy the file to the`<TOMCAT_HOME>/setup/platform_conf/licenses` folder or `<WILDFLY_HOME>/setup/platform_conf/licenses/` folder before starting the bundle.
-- If the bundle has already been started, [you need to use the Platform setup tool](BonitaBPM_platform_setup.md#update_platform_conf) to push your license file to database.
-Don't forget to restart your application server after pushing the license to the database.
+## Only once, if the application server had never been started yet: install the very first license file
+
+When you receive the license file (`.lic` file extension), store the file into the `<TOMCAT_HOME>/setup/platform_conf/licenses` folder or `<WILDFLY_HOME>/setup/platform_conf/licenses/` folder before starting the bundle.
+
+## Otherwise, Save a new license file in the database; when the application server has already been started successfuly once (or more)
+
+- Download the configuration on the disk
```

Comment:
```suggestion
If the application server has already been started successfully at least once and you need to install a new license because the one already installed has expired or is no longer valid:
- Download the configuration on the disk
```